eng/Versions.props

```diff
@@ -28,6 +28,10 @@
     <MicrosoftDotNetBuildTasksFeedPackageVersion>5.0.0-beta.19515.1</MicrosoftDotNetBuildTasksFeedPackageVersion>
     <MicrosoftDotNetBuildTasksPackagingPackageVersion>5.0.0-beta.19515.1</MicrosoftDotNetBuildTasksPackagingPackageVersion>
     <MicrosoftDotNetVersionToolsTasksPackageVersion>5.0.0-beta.19515.1</MicrosoftDotNetVersionToolsTasksPackageVersion>
+    <MicrosoftDotNetXUnitExtensionsPackageVersion>5.0.0-beta.19515.1</MicrosoftDotNetXUnitExtensionsPackageVersion>
+    <!-- Testing -->
+    <XUnitPackageVersion>2.4.1</XUnitPackageVersion>
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.0.0-preview6-27804-01</MicrosoftDotNetPlatformAbstractionsPackageVersion>
```

Comment:
Why this version? Can you use the stable 3.0.0 instead?

Reply:
I copied it [from CoreFx](https://github.com/dotnet/corefx/blob/c98870374325bb3e55cf764757826ba574d7186a/eng/Versions.props) and thought bot will take it from here. Updated.

Reply:
Ah, bot only takes care of things that are depended on in `eng/Version.Details.xml` (and only when there's a newly produced build), so it wouldn't do anything with this.

Would be *nice* to update the CoreFX version, but may break something I suppose... in this PR's case, it'll be good to use a stable version to start with.